DevEdu.API/DevEdu.DAL/Repositories/UserRepository.cs

```diff
@@ -1,12 +1,88 @@
-﻿using System;
+﻿using DevEdu.DAL.Models;
+using Dapper;
+using System.Data;
+using System.Data.SqlClient;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace DevEdu.DAL.Repositories
 {
     public class UserRepository
     {
+        string connectionString = "Data Source=80.78.240.16;Initial Catalog = DevEdu; Persist Security Info=True;User ID = student;Password=qwe!23; Pooling=False;MultipleActiveResultSets=False;Connect Timeout = 60; Encrypt=False;TrustServerCertificate=False";
+
+        private readonly IDbConnection _connection;
+
+        public UserRepository()
+        {
+            _connection = new SqlConnection(connectionString);
+        }
+
+        public int AddUser(UserDto user)
+        {
+            return _connection.QuerySingle<int>(
+                "dbo.User_Insert", 
```

Comment:
each name of stored procedures must be placed somewhere as a constant
otherwise, it's an example of the "magic numbers" antipattern